ramp-frontend/ramp_frontend/views/ramp.py

```diff
@@ -428,6 +428,8 @@ def sandbox(event_name):
                         src=os.path.join(sandbox_submission.path, filename),
                         dst=os.path.join(new_submission.path, filename)
                     )
+                # we delayed the state of the submission until that we copy the file
+                new_submission.set_state('new')
```

Comment:
This function does not commit in the database. If you could either call `db.session.commit()` or use `ramp_database.tools.set_submission(db.session, new_submission.id, state)

Reply:
Could you do that on your server as well to check if it solves the issue?